mods/default/functions.lua

```diff
@@ -719,7 +719,7 @@ end
 -- Log API / helpers
 --
 
-local log_non_player_actions = minetest.settings:get_bool("log_non_player_actions") ~= false
+local log_non_player_actions = minetest.settings:get_bool("log_non_player_actions") or false
```

Comment:
Isn't this superfluous code? The docs for `get_bool` state the following:

> `default` is the value returned if `key` is not found

Considering that the default `false` is already set in `settingtypes.txt`, you don't need to specify it here.

Reply:
`minetest.settings:get_bool("log_non_player_actions")` returns `nil` if not set. So i want to make sure that the value is a bool

Reply:
ah sorry, now i see what you mean and fix this ....

Reply:
fixed in https://github.com/minetest/minetest_game/pull/2941/commits/fc6e5944a21a3ec4c36310cb3de8ef8caf4f8e91